<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="./styles/style.css">
    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
    <link rel="icon" type="image/png" href="./Plants-Sproud-icon-transparent.png">
    <title>Dr. Rebecca Batstone / Research</title>
</head>

<body>
    <header>
        <div class="wrapper">
            <!--MAIN NAVIGATION-->
            <div class="header">
                <img src="./assets/roots-removebg-preview.png" alt="" class="header__img">
                <button class="header__mobile-icon-container"><i class="fas fa-bars js-menu header__mobile-icon"
                        title="Hamburger Menu"></i></button>
                <span class="sr-only">Hamburger icon to trigger mobile navigation</span>
                <nav class="header__nav">
                    <ul class="header__nav__list" id="js-nav">
                        <li class="header__nav__list__item"><a href="./index.html">Home</a></li>
                        <li class="header__nav__list__item header__nav__list__link"><a
                                href="./research.html">Research</a>
                        </li>
                        <li class="header__nav__list__item header__nav__list__link"><a
                                href="./teaching.html">Teaching/Outreach</a></li>
                        <li class="header__nav__list__item header__nav__list__link"><a href="./talks.html">Talks</a>
                        </li>
                    </ul>
                </nav>
            </div>
        </div>
        <ul class="header__social-list">
            <li><a href="https://twitter.com/BatstoneRebecca" target="_blank"><i class="fab fa-twitter"></i></a>
            </li>
            <li><a href="https://www.instagram.com/rebeccabatstone/" target="_blank"><i
                        class="fab fa-instagram"></i></a>
            </li>
            <li><a href="https://github.com/rtbatstone" target="_blank"><i class="fab fa-github"></i></a>
            </li>
            <li><a href="mailto:rtbatstone@gmail.com"><i class="fas fa-envelope-square"></i></a></li>
        </ul>
    </header>

    <main>

        <section class="dark js--main-intro">
            <div class="wrapper">
                <div class="post">
                    <h2 class="post-title-center">Darwin's hidden bank - the microbiome</h2>
                    <div class="post-1">
                        <div class="post-1__container-1">
                            <div class="post-1__container-1__text-1">
                                <i class="fas fa-quote-left"></i>
                                <p>"It is interesting to contemplate a tangled bank, clothed with many plants of many
                                    kinds,
                                    with birds singing on the bushes, with various insects flitting about, and with
                                    worms
                                    crawling through the damp earth, and to reflect that these elaborately constructed
                                    forms, so
                                    different from each other, and dependent upon each other in so complex a manner,
                                    have
                                    all
                                    been produced by laws acting around us.” - Darwin, 1859</p>
                            </div>
                            <div class="post-1__container-1__img-1 img-container">
                                <img src="./assets/research/microbiome_orig.png" alt="Microbes on and within humans.">
                                <p class="sub-text">Kateryana Kon/Science Photo Library, via Getty Images</p>
                            </div>
                        </div>
                        <div class="post-1__container-2">
                            <p>Darwin's tangled bank paints a picture of a "macrobial" world - one that's predominantly
                                inhabited by plants, humans, and other animals. Since the dawn of the Next-Generation
                                Sequencing era, however, it’s never been more clear that "macrobes" are actually
                                newcomers in a microbial world, having (co)evolved with microbes since the beginning of
                                their comparatively short evolutionary history. Many microbes, including bacteria,
                                archaea, fungi, and viruses, are symbiotic, living on or within macrobial host tissues.
                                Importantly, symbiotic microbes have the capacity to generate novel extended phenotypes
                                that play vital roles in host fitness by affecting nutrient availability and uptake,
                                resistance to pathogens and other host enemies, and even host behavior. Thus, symbiotic
                                microbial genomes encode important, though often unknown, sources of genetic variation
                                that potentially facilitate host adaptation. While Darwin and subsequent biologists have
                                described the laws of evolution relevant to a macrobial world, advances in microbiology
                                and molecular genetics call into question whether these laws need to be extended, or
                                even rewritten, in a microbial and symbiotic world.</p>
                            <p>​My research program leverages classical evolutionary theory, population genomics,
                                microbiology, experimental evolution, and bioinformatics to predict evolution when we
                                explicitly account for symbiosis. Below I describe how my past work has used a variety
                                of empirical approaches to address critical questions that drive my research program,
                                then outline how I will expand this work in the future in my own lab.</p>
                        </div>
                    </div>

                </div>
            </div>
        </section>

        <section class="light">
            <div class="wrapper">
                <div class="post" data-aos="fade-up">
                    <div class="post-2">
                        <div class="post-2__container-1">
                            <h2>Using the legume-rhizobium symbiosis as a focal interaction within a microbiome to
                                better
                                understand host-microbe coevolution</h2>
                            <p>Many people depend on leguminous plants such as beans and peas as a major source of their
                                daily
                                protein. But what some might not know is that legumes are nitrogen rich because of their
                                symbiotic relationship with bacteria. Bacteria, known as rhizobia, dwell in the soil for
                                part of
                                their life cycle, but it’s only when they intimately associate with a legume via
                                symbiosis
                                that
                                they begin to fix nitrogen. The plant and its rhizobia partners enter in to a mutual
                                exchange of
                                resources – the plant provides carbon in the form of sugar to rhizobia. Reciprocally,
                                rhizobia
                                fix atmospheric nitrogen into usable forms for the plant, such as ammonia. The site of
                                nutrient
                                exchange takes place within nodules located on the legume’s roots, each nodule typically
                                housing
                                a single strain of rhizobia (Fig. 1A below). </p>
                            <p>This symbiosis contributes 70% of all biologically fixed N to the global N cycle, and
                                thus,
                                represents one of the most ecologically and economically valuable symbiotic
                                interactions. It
                                is
                                also a relatively tractable system with many genomic and molecular resources available,
                                making
                                it an ideal model for understanding symbiosis evolution.</p>
                            <div class="post-2__container-1__img-1 img-container">
                                <img src="./assets/research/Legume_rhizobia_interaction.png"
                                    alt="Exchange of resources between legumes and rhizobiam">
                                <p class="sub-text">Fig. 1A: resource exchange between legumes and rhizobia</p>
                            </div>
                        </div>
                        <div class="post-2__container-2">
                            <div class="post-2__container-2__img-1 img-container">
                                <img src="./assets/research/tattoo_transparent.png"
                                    alt="Medicago truncatula, common name the barrel medic.">
                                <p class="sub-text">Illustration by Lu Loram Martin</p>
                            </div>
                            <div class="post-2__container-2__text-1">
                                <p>Rhizobia genomes are especially interesting because they are multipartite, meaning
                                    they
                                    include a large chromosome as well as one or more symbiosis plasmids that can be
                                    considered
                                    distinct biological entities (Fig. 1B). Many of the genes involved in symbiotic
                                    functions,
                                    including nodulation and nitrogen fixation, are located on the plasmids. </p>
                                <div class="post-2__container-2__img-2 img-container">
                                    <img src="./assets/research/multipartite_genome.png"
                                        alt="Rhizobium genomes are multipartite: many of the genes involved in symbiosis are located on Mega-plasmids that can constitute almost 50% of the cell's entire genome. ">
                                    <p class="sub-text">Fig. 1B: Rhizobium genomes are multipartite: many of the genes
                                        involved in symbiosis
                                        are
                                        located on Mega-plasmids that can constitute almost 50% of the cell's entire
                                        genome.
                                    </p>
                                </div>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>


        <section class="dark">
            <div class="wrapper">
                <div class="post" data-aos="fade-up">
                    <h2 class="post-title-center">Combining large-scale manipulative experiments with quantitative
                        genetics
                        to understand
                        symbiosis evolution and stability</h2>
                    <div class="post-3">
                        <div class="post-3__container-1">
                            <p>Empirical evidence robustly supports widespread fitness alignment within the
                                legume-rhizobium
                                symbiosis, meaning that more effective N-fixing rhizobia receive greater fitness rewards
                                from their host plant relative to ineffective N-fixing rhizobia. Traits that allow a
                                host to
                                preferentially reward or associate with more beneficial symbionts (i.e., preference
                                traits)
                                are largely responsible for this alignment, yet we know little about how these traits
                                evolve. Using an evolutionary quantitative genetics approach in the model legume
                                <span>Medicago truncatula</span> and effective and ineffective strains of its
                                <span>Ensifer meliloti</span>symbiont, I found that plants were under selection to favor
                                the more effective N-fixer, rather than to punish or exclude the ineffective N-fixer (<a
                                    href="https://nph.onlinelibrary.wiley.com/doi/full/10.1111/nph.14308"
                                    target="_blank">Batstone et al. 2017</a>). Thus, the presence of
                                effective
                                mutualistic symbionts, rather than ineffective or ‘cheating’ symbionts, provide the
                                selective incentive for host preference traits to evolve, ultimately maintaining
                                symbiosis
                                stability.​​
                            </p>
                        </div>
                        <div class="post-3__container-2">
                            <div class="post-3__container-2__img-1">
                                <img src="./assets/research/split-root.png"
                                    alt="Split-root set up, each root compartment either containing different concentrations of N-fertilizer, or strains of rhizobia that fix different levels of N.">
                            </div>
                            <div class="post-3__container-2__img-2 img-container">
                                <img src="./assets/research/split-root-full.png"
                                    alt="Split-root set up, each root compartment either containing different concentrations of N-fertilizer, or strains of rhizobia that fix different levels of N.">
                                <p class="sub-text">Split-root set up, each root compartment either containing different
                                    concentrations of
                                    N-fertilizer, or strains of rhizobia that fix different levels of N.</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>

        <section class="light">
            <div class="wrapper">
                <div class="post" data-aos="fade-up">
                    <div class="post-4">
                        <h2>Experimental evolution coupled with bioinformatics uncovers the
                            genomic
                            underpinnings of
                            microbial adaptation within symbiosis</h2>
                        <div class="post-4__container-1">
                            <div class="post-4__container-1__img-1">
                                <img src="./assets/research/Batstone_approaches.png"
                                    alt="The novel approaches used to study microbial symbiosis.">
                            </div>
                        </div>
                        <div class="post-4__container-2">
                            <div class="post-4__container-2__img-1 img-container">
                                <img src="./assets/research/experimental-evolution.jpg"
                                    alt="The experimental evolutino approach.">
                                <p class="sub-text">Photo credit: Bill Cole</p>
                            </div>
                            <div class="post-4__container-2__img-2 img-container">
                                <img src="./assets/research/experimental-evo.jpeg" alt="Dr. Rebecca Batstone.">
                                <p class="sub-text">Photo credit: Megan Frederickson</p>
                            </div>
                        </div>
                        <div class="post-4__container-3">
                            <p>In a recent publication in <a
                                    href="https://science.sciencemag.org/content/370/6515/476.abstract"
                                    target="_blank">Science</a>, we explored how beneficial microbes adapt to
                                hosts. Advances in microbiome science require a better understanding of how beneficial
                                microbes
                                adapt to hosts. Mutualism theory predicts that choosy hosts should select for microbial
                                partners that maximize host fitness (i.e., high-quality partners). As high-quality
                                partners
                                become fixed in a population, however, host preference paradoxically erodes the genetic
                                variation in microbial quality required for host preference. Yet, variation in microbial
                                quality is extensive in nature, begging the question: what factors maintain variation in
                                partner quality? By combining a year-long evolution experiment, a cross-inoculation
                                experiment, and whole genome resequencing, I overwhelmingly found that that variation in
                                microbial quality was maintained by rapid adaptation of symbionts to their local host:
                                isolates that shared an evolutionary history with a particular host genotype contributed
                                more to that host's growth and achieved higher fitness than when paired with another
                                host.
                                Local adaptation resulted from genomic variants that had host-specific (i.e.,
                                conditionally-neutral) effects, rather than variants that were beneficial on one host
                                but
                                detrimental on another (i.e., antagonistic pleiotropy). Overall, these results suggest
                                microbial adaptation is an important driver of genetic variation in microbial quality,
                                given
                                that adapting to one host genotype causes less favorable (but not necessarily
                                deleterious)
                                outcomes on another host genotype.</p>
                        </div>
                    </div>
                </div>
            </div>
        </section>

        <section class="dark">
            <div class="wrapper">
                <div class="post" data-aos="fade-up">
                    <h2>Symbiosis evolution under changing environmental conditions</h2>
                    <div class="post-5">
                        <div class="post-5__container-1">
                            <div class="post-5__container-1__img-1 img-container">
                                <img src="./assets/research/lter-system.png"
                                    alt="Long term ecological research (LTER) plots located at the Kellogg Biological Station in Michigan. For the past 30 years, half of the experimental plots have been given additional N fertilizer, while the other half have not, allowing you to compare plant or soil communities between N-treatments.">
                                <p class="sub-text">Long term ecological research (LTER) plots located at the Kellogg
                                    Biological Station in
                                    Michigan. For the past 30 years, half of the experimental plots have been given
                                    additional N fertilizer, while the other half have not, allowing one to compare
                                    plant or
                                    soil communities between N-treatments</p>
                            </div>
                            <div class="post-5__container-1__img-2 img-container">
                                <img src="./assets/research/trifolium-evo.png"
                                    alt="Experimentally evolving rhizobium under high and low-N conditions to recapitulate evolution occurring in the field. Pooled and whole-genome sequencing will be used to quantify relative strain fitness in each community type and N-treatment.">
                                <p class="sub-text">Experimentally evolving rhizobium under high and low-N conditions to
                                    recapitulate
                                    evolution occurring in the field. Pooled and whole-genome sequencing will be used to
                                    quantify relative strain fitness changes in allele frequencies, and prevalence of
                                    horizontal gene transfer in each community type and N-treatment.</p>
                            </div>
                        </div>
                        <div class="post-5__container-2">
                            <h3>Rhizobial adaptation to N-fertilization</h3>
                            <p>Theory predicts that legumes should reduce symbiotic investment when abiotic N is
                                abundant,
                                given that symbiotically-derived N is more costly to acquire than N from the soil.
                                Indeed,
                                long-term N-fertilization was found to select for low-quality rhizobia (<a
                                    href="https://onlinelibrary.wiley.com/doi/abs/10.1111/evo.12594"
                                    target="_blank">Weese et al.
                                    2015</a>),
                                and the genetic basis for this shift occurred in mobile symbiosis plasmids (<a
                                    href="https://royalsocietypublishing.org/doi/full/10.1098/rspb.2015.2563"
                                    target="_blank">Klinger et
                                    al.</a>
                                2016). However, the relative roles of standing genetic variation, novel mutations, and
                                horizontal gene transfer (HGT) in rhizobial adaptation to high-N conditions remain
                                unknown.
                                I am currently conducting a large-scale evolution experiment that tests how
                                N-fertilization
                                affects rhizobial population composition and evolution over time. Specifically, I
                                competed
                                N-adapted and non-N-adapted (i.e., control) strains in single and mixed populations
                                under
                                high and low levels of N and across multiple plant populations (schematic above). By
                                measuring strain frequencies in each population over time, I can test whether N-adapted
                                strains outcompete control strains only under high-N, and whether N-adaptation is costly
                                under N-limited conditions. I will also sequence the entire genomes of the strains at
                                the
                                end of the experiment to test how allele frequencies are impacted by N-level, and
                                whether
                                HGT occurred, given that many of the rhizobial genes involved in symbiosis are located
                                on
                                mobile plasmids or islands.</p>
                        </div>
                    </div>
                </div>
        </section>

        <section class="light">
            <div class="wrapper">
                <div class="post" data-aos="fade-up">
                    <div class="post-6">
                        <div class="post-6__container-1">
                            <h2>Future directions</h2>
                            <h3>Treating the legume-rhizobium symbiosis as a nested interaction within a complex
                                phytobiome</h3>
                            <p>The legume-rhizobium symbiosis has proven to be a powerful model for understanding how
                                mutualistic microbial symbioses evolve and respond to changing environmental conditions.
                                Using the toolkit I've built while working on this system, I will address the role of
                                horizontal gene transfer (HGT) in natural rhizobia populations. It is currently unknown
                                how
                                frequently HGT occurs in nature, and what environmental cues might trigger it. I will
                                test
                                the role of HGT in invasion success, and specifically whether rhizobia can acquire
                                plasmids
                                or genes that help them spread across newly invaded sites. Rather than occurring in a
                                vacuum, however, this symbiosis is embedded within a complex community of interacting
                                microbes, the phytobiome. Moving forward, I will treat the legume-rhizobium symbiosis as
                                a
                                focal interaction within a phytobiome, ideally conducting similar experimental evolution
                                approaches as described above, but within the context of naturally-occurring
                                microbiomes.
                                Overall, this work would help pinpoint the genetic mechanisms underlying microbial
                                adaptation to stressful conditions, and what effects microbial evolution have on host
                                fitness.</p>
                        </div>
                        <div class="post-6__container-2">
                            <div class="post-6__container-2__img-1 img-container">
                                <img src="./assets/research/Nested_interactions.png"
                                    alt="The legume-rhizobium symbiosis represents multiple nested layers of interaction. Mobile genetic elements nested within microbes, nested within macrobes.">
                            </div>
                            <p class="sub-text">The legume-rhizobium symbiosis represents multiple nested layers of
                                interaction. 1) Plasmids,
                                or other mobile genetic elements, infect a rhizobium cell. 2) Once a rhizobium cell is
                                infected by the ‘right’ plasmid, it can then infect a plant root. The interaction
                                between a
                                rhizobium chromosome and plasmid(s) is, thus, necessary to form the symbiosis with the
                                legume, the second layer of the nested interaction. 3) Rhizobia divide and proliferate
                                within nodules that eventually break open or senesce, releasing rhizobia into the soil.
                                Here, rhizobia switch to a free-living lifestyle and are thought to exchange or acquire
                                new
                                plasmids, potentially giving them the ability to infect a different host. Understanding
                                how
                                these multiple distinct entities form nested interactions that produce the emergent
                                traits
                                we care about – for example, plant yield, is further complicated by the fact that they
                                evolve within a tangled web of interactions involving millions of other microbes.</p>
                        </div>
                    </div>
                </div>
            </div>
        </section>

    </main>

    <!--FOOTER-->
    <footer class="footer">
        <div class="wrapper">
            <div class="footer__container">
                <div class="about__social-container">
                    <ul class="footer__social-list">
                        <li><a href="https://twitter.com/BatstoneRebecca" target="_blank"><i
                                    class="fab fa-twitter"></i></a></li>
                        <li><a href="https://www.instagram.com/rebeccabatstone/" target="_blank"><i
                                    class="fab fa-instagram"></i></a>
                        </li>
                        <li><a href="https://github.com/rtbatstone" target="_blank"><i class="fab fa-github"></i></a>
                        </li>
                        <li><a href="mailto:rtbatstone@gmail.com"><i class="fas fa-envelope-square"></i></a></li>
                    </ul>
                </div>
                <nav class="footer-nav">
                    <ul class="footer__nav-list">
                        <li class="footer__nav-link"><a href="./index.html">Home</a></li>
                        <li class="footer__nav-link"><a href="./research.html">Research</a></li>
                        <li class="footer__nav-link"><a href="./teaching.html">Teaching/Outreach</a></li>
                        <li class="footer__nav-link"><a href="./talks.html">Talks</a></li>
                    </ul>
                </nav>
                <a href="https://www.adambatstone.dev/" target="_blank" class="footer__container__design">&copy; 2020
                    BSTONE Web
                    Design</a>
            </div>
        </div>
    </footer>

    <script src="https://kit.fontawesome.com/ae5be4dc98.js" crossorigin="anonymous"></script>
    <script src="https://code.jquery.com/jquery-2.2.4.min.js"
        integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44=" crossorigin="anonymous"></script>
    <script src="./scripts/jquery.waypoints.min.js"></script>
    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
    <script src="./scripts/script.js"></script>
</body>

</html>